src/components/Header/Search/SearchInput/SearchInput.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SearchCSS from './SearchInput.module.css';
+import PropTypes from "prop-types";
+
+const SearchInput = (props) => (
+    <input className={SearchCSS.main} placeholder={props.title}>
+    </input>
```

Comment:
In general "input" doesn't have the end tag

Reply:
Fixed